pftools/python/parflow/tools/core.py

```diff
@@ -512,6 +514,39 @@ def run(self, working_directory=None, skip_validation=False):
         if not success or error_count > 0:
             sys.exit(1)
 
+    def check_nans(self, working_directory, exclude_forcing=True):
```

Comment:
An API best practice I picked up somewhere is to avoid negative flags.   Suggest "include_forcing".

Generally makes the code a bit easier to read ("if include_forcing" without the 'not') and documentation reads generally reads easier "If set to True, forcing files will be checked" if you don't have to do a mental true/false inversion.

Reply:
Thanks for the suggestion, makes sense.